### PR TITLE
feat: added option menu

### PR DESCRIPTION
**Pull request by SolMendiola**
# :star: Feature

---

#### :pencil2: Description:

Added option menu

---

#### iOS

https://github.com/xmartlabs/supabase-ws-flutter/assets/57235692/ca3d4e02-30b8-4a90-9041-28ad562a12fa

### CODE REVIEW

File: mobile/lib/core/common/config.dart [hunk 1]
```diff
@@ -10,7 +10,9 @@ import 'package:flutter_template/core/common/helper/env_helper.dart';
 
 interface class Config {
   static const String environmentFolder = 'environments';
-
+  static const String gitHubUri =
```
Comment (nicolantean) [inline mobile/lib/core/common/config.dart:13]: Url maybe

File: mobile/lib/core/common/config.dart [hunk 2]
```diff
@@ -10,7 +10,9 @@ import 'package:flutter_template/core/common/helper/env_helper.dart';
...
+      'https://github.com/xmartlabs/supabase-ws-flutter';
+  static const String xmartLabsUri = 'https://xmartlabs.com/';
```
Comment (nicolantean) [inline mobile/lib/core/common/config.dart:15]: same

File: mobile/lib/ui/home/home_screen.dart [hunk 3]
```diff
@@ -28,6 +30,7 @@ class _HomeContentScreen extends StatefulWidget {
 
 class _HomeContentScreenState extends State<_HomeContentScreen> {
   final _textController = TextEditingController();
+  OptionsMenu? selectedMenu;
```
Comment (nicolantean) [inline mobile/lib/ui/home/home_screen.dart:33]: why do you need to save this? it's just an option menu with actions when pressing the different options

File: mobile/lib/ui/home/home_options_menu.dart [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+enum _OptionsMenu {
+  github,
+  xmartlabs,
+  logout;
+}
+
+extension _OptionsMenuExtension on _OptionsMenu {
+  String get name => switch (this) {
```
Comment (mirland) [inline mobile/lib/ui/home/home_options_menu.dart:76]: context

**Review by mirland — APPROVED** [review]

**Review by cecipirotto — APPROVED** [review]

**Pull request merged**